src/card.js

```diff
@@ -0,0 +1,53 @@
+const card = function card() {
```

Comment:
- [ ] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

   * [ ] usage of arrow functions instead of the `function` keyword
```suggestion
const card = () => {
```